Makefile

```diff
@@ -4,6 +4,13 @@ HELM_PLUGIN_DIR ?= $(HELM_HOME)/plugins/helm-2to3
 VERSION := $(shell sed -n -e 's/version:[ "]*\([^"]*\).*/\1/p' plugin.yaml)
 DIST := $(CURDIR)/_dist
 LDFLAGS := "-X main.version=${VERSION}"
+MOD_PROXY_URL ?= https://gocenter.io
+
+PHONY: bootstrap
+bootstrap:
+	export GO111MODULE=on && \
+    export GOPROXY=$(MOD_PROXY_URL) && \
```

Comment:
Do you mins aligning with other `bootstrap` commands?

Reply:
sure, that VS Code show is differently

Reply:
done